resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -294,3 +294,13 @@ In addition, you can run a single test selectively.
         -Dspark.kubernetes.test.deployMode=docker-desktop \
         -Dspark.kubernetes.test.imageTag=2022-03-06 \
         'kubernetes-integration-tests/testOnly -- -z "Run SparkPi with a very long application name"'
+
+You can also specify your specific dockerfile to build JVM/Python/R based image to test.
+
+    build/sbt -Psparkr -Pkubernetes -Pkubernetes-integration-tests \
+        -Dtest.exclude.tags=minikube \
+        -Dspark.kubernetes.test.deployMode=docker-desktop \
+        -Dspark.kubernetes.test.DockerFile=/path/to/Dockerfile \
+        -Dspark.kubernetes.test.pyDockerFile=/path/to/Dockerfile.py \
+        -Dspark.kubernetes.test.rDockerFile=/path/to/Dockerfile.r \
```

Comment:
ditto. `Dockerfile.r` looks like R script. It should not.

Reply:
/path/to/r/Dockerfile